src/Stryker.Core/Stryker.Core.UnitTest/Options/Inputs/IgnoreMutationsInputTests.cs

```diff
@@ -0,0 +1,23 @@
+using Shouldly;
+using Stryker.Core.Exceptions;
+using Stryker.Core.Options.Inputs;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Options.Inputs
+{
+    public class IgnoreMutationsInputTests
+    {
+        [Fact]
+        public void ShouldValidateExcludedMutation()
+        {
+            var target = new IgnoreMutationsInput();
+            target.SuppliedInput = new[] { "gibberish" };
+
+            var ex = Should.Throw<StrykerInputException>(() =>
+            {
+                target.Validate();
+            });
+            ex.Message.ShouldBe($"Invalid excluded mutation (gibberish). The excluded mutations options are [Arithmetic, Equality, Boolean, Logical, Assignment, Unary, Update, Checked, Linq, String, Bitwise, Initializer, Regex]");
```

Comment:
Maybe it's worth generating this list separately (and testing the generation as well) as this can easily and silently go out of sync.